TSConfckParseError relating to docusaurus in website when building UI package

### Describe the bug

I get non-fatal errors when trying to `npm run build` inside the UI package



### To Reproduce

```
cd packages/ui
npm run build
```

### Expected behavior

shouldn't be errors relating to a website directory inside a UI package build.

### Your Environment

```markdown
- pdfme package: @pdfme/ui
- pdfme version: latest 3
- Operating system: Mac OSX
- Node.js version: 18.17.0
```


### Your Error Log

```shell
[tsconfig-paths] An error occurred while parsing "/Users/pete/Projects/pennyblack/pdfme-upstream/website/tsconfig.json". See below for details. To disable this message, set the `ignoreConfigErrors` option to true.
_TSConfckParseError: failed to resolve "extends":"@tsconfig/docusaurus/tsconfig.json" in /Users/pete/Projects/pennyblack/pdfme-upstream/website/tsconfig.json
    at resolveExtends (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/tsconfck/dist/index.js:496:9)
    at parseExtends (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/tsconfck/dist/index.js:450:28)
    ... 5 lines matching cause stack trace ...
    at async build (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/vite/dist/node/chunks/dep-ErEj4WmL.js:64922:20)
    at async CAC.<anonymous> (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/vite/dist/node/cli.js:845:9) {
  code: 'EXTENDS_RESOLVE',
  cause: Error: Cannot find module '@tsconfig/docusaurus/tsconfig.json/tsconfig.json'
  Require stack:
  - /Users/pete/Projects/pennyblack/pdfme-upstream/website/tsconfig.json
      at Module._resolveFilename (node:internal/modules/cjs/loader:1077:15)
      at Function.resolve (node:internal/modules/cjs/helpers:116:19)
      at resolveExtends (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/tsconfck/dist/index.js:491:34)
      at parseExtends (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/tsconfck/dist/index.js:450:28)
      at Module.parse (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/tsconfck/dist/index.js:388:24)
      at async Promise.all (index 0)
      at async configResolved (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/vite-tsconfig-paths/dist/index.mjs:99:10)
      at async Promise.all (index 1)
      at async resolveConfig (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/vite/dist/node/chunks/dep-ErEj4WmL.js:66032:5)
      at async build (file:///Users/pete/Projects/pennyblack/pdfme-upstream/node_modules/vite/dist/node/chunks/dep-ErEj4WmL.js:64922:20) {
    code: 'MODULE_NOT_FOUND',
    requireStack: [
      '/Users/pete/Projects/pennyblack/pdfme-upstream/website/tsconfig.json'
    ]
  },
  tsconfigFile: '/Users/pete/Projects/pennyblack/pdfme-upstream/website/tsconfig.json'
}
```
```


### Additional context

_No response_

--- a/packages/ui/vite.config.ts
+++ b/packages/ui/vite.config.ts
@@ -6,7 +6,7 @@ import topLevelAwait from 'vite-plugin-top-level-await';
 export default defineConfig({
   plugins: [
     react(),
-    tsconfigPaths(),
+    tsconfigPaths({root: '.'}),
     topLevelAwait({
       promiseExportName: '__tla',
       promiseImportName: (i) => `__tla_${i}`,

